src/chapter-03/src/core.rs

```diff
@@ -1,19 +1,7 @@
 use cuid2::CuidConstructor;
-use lazy_static::lazy_static;
-use rand::distributions::Alphanumeric;
-use rand::{thread_rng, Rng};
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
-use std::sync::Mutex;
-
-// This is for DEMONSTRATION purposes only
-// Static variables are normally a bad practice in Lambda, as separate execution environments will not share values
-lazy_static! {
```

Comment:
I prefer to avoid `lazy_static` (and static variables in general) because they make testing very hard...
I realised that here we already have a shared instance of `UrlShortener` so it makes sense to move the state inside that struct